website/index.html

```diff
@@ -0,0 +1,294 @@
+</head>
+<body>
+  <header class="bg-white">
+    <div class="sm:flex sm:justify-between sm:items-center sm:py-4 relative z-10 max-w-screen-lg xl:max-w-screen-xl mx-auto">
+      <div class="flex items-center justify-between px-4 py-2 sm:p-0">
+        <div>
+          <img class="h-12 sm:hidden" src="fe-logo-small.svg" alt="Fe Programming Language">
+        </div>
+        <div class="sm:hidden">
+          <button id="menu-btn" type="button" class="block text-gray-600 hover:text-black focus:text-black">
+            <svg class="h-6 w-6 fill-current" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+              <path id="menu-icon" class=" block" troke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+              <path id="close-icon" class="hidden" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+            </svg>
+          </button>
+        </div>
+      </div>
+      <nav id="menu" class="hidden px-2 pt-2 pb-4 text-center sm:flex sm:p-0 sm:text-left">
+        <a href="docs/quickstart/index.html" class="transition duration-100 ease-in-out block text-lg px-2 py-2 hover:bg-gray-700 hover:text-white rounded-sm sm:mt-0 sm:ml-2 sm:px-4" title="Get started" >Get started</a>
+        <a href="docs/index.html" class="transition duration-100 ease-in-out mt-1 text-lg block px-2 py-2 hover:bg-gray-700 hover:text-white rounded-sm sm:mt-0 sm:ml-2 sm:px-4" title="Learn">Learn</a>
+        <a href="https://discord.gg/ywpkAXFjZH" class="transition duration-100 ease-in-out mt-1 text-lg block px-2 py-2 hover:bg-gray-700 hover:text-white rounded-sm sm:mt-0 sm:ml-2 sm:px-4"title="Community">Community</a>
+        <a href="https://snakecharmers.ethereum.org/tag/fe/" class="transition duration-100 ease-in-out mt-1 text-lg block px-2 py-2 hover:bg-gray-700 hover:text-white rounded-sm sm:mt-0 sm:ml-2 sm:px-4" title="Blog">Blog</a>
+      </nav>
+    </div>
+  </header> 
+  <main role="main">
+    <section class="px-4 py-12 pt-12 sm:py-16 lg:pt-24 max-w-screen-lg xl:max-w-screen-xl mx-auto md:flex justify-between lg:justify-around items-center bg-red-500 rounded-sm">
+      <img class="h-52 lg:h-72 lg:ml-8 xl:ml-0 hidden sm:block" src="fe-logo.svg" alt="Fe Programming language">
+      <div class="sm:mt-14 md:ml-16 md:mt-0 max-w-3xl lg:max-w-2xl xl:max-w-3xl xl:-ml-4">
+        <h2 class="text-5xl sm:text-6xl xl:text-6xl leading-none font-semibold tracking-tight">The <span class="text-gray-100">next generation</span><br> smart contract language for Ethereum</h2>
+        <p class="text-lg sm:text-2xl text-gray-700 leading-6 sm:leading-8 mt-4">Create <span class="text-gray-900 font-semibold">decentralized applications</span> in a powerful, future-proof and <span class="text-gray-900 font-semibold">statically typed</span> language that is very <span class="text-gray-900 font-semibold">easy to learn</span>.<p>
```

Comment:
IMO the `text-gray-700` and `text-gray-900` colors are a bit to close. Also, I think we want 900 to pop a bit more, so maybe that one should be the lighter shade.

Reply:
I increased the contrast between them but switching them didn't look good to me. Probably because then we would not have the contrast to the headline above. But if truly suck at design work so if there is potential for optimizations I would ask Pascal post-merge to give it a make over.

Reply:
looks good :+1: